calculatorApp/README.md

```diff
@@ -0,0 +1,32 @@
+# Note
+I built this calculator for the 2020-04-01 MintBean Hackathon. While doing this I used a tutorial to build the logic of the calculator. Learned a lot about switch statements and input binding!
+
+link to tutorial -  https://www.techiediaries.com/angular/angular-9-tutorial-and-example/
+
+# CalculatorApp
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.0.2.
```

Comment:
Add your deployment link somewhere at the top. Thanks for sharing the tutorial you used.